app/src/routes/v1/keycloak.js

```diff
@@ -23,4 +32,70 @@ keycloakRouter.get('/serviceClients', keycloak.protect('realm:GETOK_ADMIN'), asy
   }
 });
 
+// Creates a service client based on the configuration posted
+keycloakRouter.post('/configForm', [
+  body('configForm.applicationAcronym').isString(),
+  body('configForm.applicationName').isString(),
+  body('configForm.applicationDescription').isString(),
+  body('configForm.commonServices').isArray(),
+  body('configForm.clientEnvironment').isIn(['DEV', 'TEST', 'PROD']),
+  body('passwordPublicKey').isString()
+], async (req, res) => {
+  const userid = req.kauth.grant.access_token.content.sub;
+
+  // Validate for Bad Requests
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    log.debug('/configForm', 'Validation Error');
+    return res.status(400).json({
+      errors: errors.array(),
+      message: 'Validation failed'
+    });
```

Comment:
Use the api-problem library for any non-successful responses. In this case, we return HTTP 422 for validation errors now. Check the other routers for examples.

Reply:
Fixed